lib/normalizePath.js

```diff
@@ -17,7 +18,7 @@ export default function normalizePath(
   }
 
   if (
-    rawPath === '.' || rawPath.startsWith('./') || rawPath.startsWith('../')
+    rawPath === '.' || rawPath.startsWith('./') || rawPath.startsWith('../') || path.isAbsolute(rawPath)
```

Comment:
This doesn't seem quite right. Was this fixing something that was broken before? 🤔